docs/readmes/basics/quick_start_guide.md

```diff
@@ -154,8 +154,9 @@ After starting the Orchestrator with `HOST [magma/orc8r/cloud/docker]$ ./run.py`
 and importing `admin_operator.pfx`, you should be able to visit the Swagger UI
 at [https://localhost:9443/swagger/v1/ui](https://localhost:9443/swagger/v1/ui).
 Note that your browser may refuse to accept the server certificate from
-`localhost:9443`. Firefox and Safari will let you override this warning, but
-Chrome will not.
+`localhost:9443`. Firefox and Safari will let you override this warning. Chrome
+will also let you bypass the warning if you type "thisisunsafe". See [here](https://www.technipages.com/google-chrome-bypass-your-connection-is-not-private-message)
+for details.
```

Comment:
https://magma.github.io/magma/docs/next/contributing/contribute_conventions#:~:text=Use%20descriptive%20hyperlink,for%20a%20hyperlink

```suggestion
will also let you [bypass the warning if you type `thisisunsafe`](https://www.technipages.com/google-chrome-bypass-your-connection-is-not-private-message).
```